modules/aws-team-roles/versions.tf

```diff
@@ -6,5 +6,9 @@ terraform {
       source  = "hashicorp/aws"
       version = ">= 4.9.0"
     }
+    local = {
```

Comment:
Where is this being used?

Reply:
https://github.com/cloudposse/terraform-aws-components/blob/master/modules/aws-team-roles/outputs.tf#L6